python/raft-dask/raft_dask/test/conftest.py

```diff
@@ -1,54 +1,49 @@
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
 
 import os
 
 import pytest
 
-from dask.distributed import Client
-from dask_cuda import LocalCUDACluster, initialize
-
-os.environ["UCX_LOG_LEVEL"] = "error"
+from dask_cuda import LocalCUDACluster
 
+from raft_dask.common.utils import create_client
 
-enable_tcp_over_ucx = True
-enable_nvlink = False
-enable_infiniband = False
+os.environ["UCX_LOG_LEVEL"] = "error"
 
 
 @pytest.fixture(scope="session")
 def cluster():
-    cluster = LocalCUDACluster(protocol="tcp", scheduler_port=0)
-    yield cluster
-    cluster.close()
+    scheduler_file = os.environ.get("SCHEDULER_FILE")
+    if scheduler_file:
+        yield scheduler_file
+    else:
+        cluster = LocalCUDACluster(protocol="tcp", scheduler_port=0)
+        yield cluster
+        cluster.close()
 
 
 @pytest.fixture(scope="session")
 def ucx_cluster():
-    initialize.initialize(
-        create_cuda_context=True,
-        enable_tcp_over_ucx=enable_tcp_over_ucx,
-        enable_nvlink=enable_nvlink,
-        enable_infiniband=enable_infiniband,
-    )
-    cluster = LocalCUDACluster(
-        protocol="ucx",
-        enable_tcp_over_ucx=enable_tcp_over_ucx,
-        enable_nvlink=enable_nvlink,
-        enable_infiniband=enable_infiniband,
-    )
-    yield cluster
-    cluster.close()
+    scheduler_file = os.environ.get("SCHEDULER_FILE")
+    if scheduler_file:
+        yield scheduler_file
+    else:
+        cluster = LocalCUDACluster(
+            protocol="ucx",
+        )
+        yield cluster
+        cluster.close()
 
 
 @pytest.fixture(scope="session")
 def client(cluster):
-    client = Client(cluster)
+    client = create_client(cluster)
     yield client
     client.close()
 
 
 @pytest.fixture()
 def ucx_client(ucx_cluster):
-    client = Client(cluster)
+    client = create_client(ucx_cluster)
```

Comment:
I think this was a bug previously where we passed `cluster` instead of `ucx_cluster `. 

Will revert if this was intentional.